test/PrizePool.t.sol

```diff
@@ -121,6 +121,14 @@ contract PrizePoolTest is Test {
         assertEq(prizeToken.balanceOf(address(prizePool)), 100);
     }
 
+    function testAccountedBalance_withdrawnReserve() public {
+        contribute(100);
+        completeAndStartNextDraw(1);
+        assertGe(prizePool.reserve(), 0);
```

Comment:
Ah- good catch.  It was supposed to check that the reserve is greater than zero, so that it's actually doing something (I was too lazy to calculate the actual reserve)